adguard_cb/src/main/java/com/adguard/android/contentblocker/UserFilterActivity.java

```diff
@@ -0,0 +1,152 @@
+package com.adguard.android.contentblocker;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.*;
+import com.adguard.android.ServiceLocator;
+import com.adguard.android.service.FilterService;
+import com.adguard.android.service.PreferencesService;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserFilterActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private UserFilterAdapter userFilterAdapter;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
Here is what's missing:

1. A placeholder when the user filter is empty.
2. Import/Export features
3. Clear button